src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -88,14 +88,78 @@ private StringEscapeCharacterFormatDefinition()
             }
         }
 
+        #endregion
+        #region Keyword - Control
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.ControlKeyword)]
+        [Name(ClassificationTypeNames.ControlKeyword)]
+        [Order(After = ClassificationTypeNames.Keyword)]
```

Comment:
i'm trying to understand hte ordering here.  WHy does `Keyword - Control` need to be after Keyword.  Note: whatever the reason, just add a comment in the code.